PatientRegistrationApp/src/components/HouseholdReviewSubmit.vue

```diff
@@ -1,22 +1,162 @@
+			</v-col>
+		</v-row>
+
+		<v-row>	
+			<v-col cols="12">
+				<div class="font-weight-medium">Primary Phone: <span class="font-weight-regular">{{dataHouseholdContactInfo.primaryPhoneNumber}} 
+					({{dataHouseholdContactInfo.primaryPhoneNumberType}})</span></div>
+				<div class="font-weight-medium">Secondary Phone: <span class="font-weight-regular">{{dataHouseholdContactInfo.secondaryPhoneNumber}} 
+					({{dataHouseholdContactInfo.secondaryPhoneNumberType}})</span></div>
+				<div class="font-weight-medium">Primary E-mail: <span class="font-weight-regular">{{dataHouseholdContactInfo.primaryEmail}}</span></div>
+				<div class="font-weight-medium">Secondary E-mail: <span class="font-weight-regular">{{dataHouseholdContactInfo.secondaryEmail}}</span></div>
+				<div class="font-weight-medium">Follow-up approval: <span class="font-weight-regular">{{dataHouseholdContactInfo.approval}}</span></div>
+			</v-col>
+		</v-row>
+
+		<v-row>	
+			<v-col cols="12">
+				<v-card class="d-flex flex-wrap" flat>
+					<v-card class="pa-2" flat min-width=33%>
+						<div class="font-weight-medium primary--text">Personal Info: Household Member #1</div>
+						<div class="font-weight-medium">Name:  <span class="font-weight-regular">{{dataHouseholdPersonalInfo[0].householdFamilyName}}, 
+						{{dataHouseholdPersonalInfo[0].householdGivenName}} {{dataHouseholdPersonalInfo[0].householdSuffix}}</span></div>
+						<div class="font-weight-medium">DOB:  <span class="font-weight-regular">{{dataHouseholdPersonalInfo[0].householdBirthDate}}</span></div>
+						<div class="font-weight-medium">Gender ID:  <span class="font-weight-regular">{{dataHouseholdPersonalInfo[0].householdGender}}</span></div>
+						<div class="font-weight-medium">Race(s):  <span class="font-weight-regular">{{dataHouseholdPersonalInfo[0].householdRaceSelections}}</span></div>
+						<div class="font-weight-medium">Ethnicity:  <span class="font-weight-regular">{{dataHouseholdPersonalInfo[0].householdEthnicitySelection}}</span></div>
+						<div class="font-weight-light primary--text">Emergency Contact</div>
+						<div class="font-weight-medium">Name:  <span class="font-weight-regular">{{dataHouseholdEmergencyContact.householdEmergencyContactFamilyName}}, 
+						{{dataHouseholdEmergencyContact.householdEmergencyContactGivenName}}</span></div>
+						<div class="font-weight-medium">Phone:  <span class="font-weight-regular">{{dataHouseholdEmergencyContact.householdEmergencyContactPhoneNumber}} 
+						({{dataHouseholdEmergencyContact.householdEmergencyContactPhoneNumberType}})</span></div>
+					</v-card>
 			
+					<v-card 
+						v-for="n in numberOfHouseholdMembers-1"
+						:key="n"
+						class="pa-2" flat min-width=33%>
+						<div class="font-weight-medium primary--text">Personal Info: Household Member #{{n+1}}</div>
+						<div class="font-weight-medium">Name:  <span class="font-weight-regular"><!--{{dataHouseholdPersonalInfo[n].householdFamilyName}}, 
```

Comment:
Can you clarify the commented out code - this is what you meant which will be populated in MVS-385?

Reply:
Yes, exactly